flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/deduplicate/ProcTimeDeduplicateKeepLastRowFunction.java

```diff
@@ -33,26 +36,50 @@
     private final boolean generateUpdateBefore;
     private final boolean generateInsert;
     private final boolean inputIsInsertOnly;
+    /** function used to equal RowData. */
+    private transient RecordEqualiser equaliser;
```

Comment:
minor: The record equaliser used to equal RowData.

we can put the transient variable to the end of all members